distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -2607,7 +2620,7 @@ private String getCswQueryResponse(String metacardId, OffsetDateTime modifiedTim
         String modifiedTime = modifiedTimestamp.format(DateTimeFormatter.ofPattern(
                 "yyyy-MM-dd'T'HH:mm:ss.SSSXXX"));
 
-        return Library.getFileContent("/csw-query-response.xml",
+        return getFileContent("/csw-query-response.xml",
```

Comment:
Looks like it's safe to do https://docs.oracle.com/javase/8/docs/api/java/lang/ClassLoader.html#getResource(java.lang.String)